kernel/realsense/d4xx.c

```diff
@@ -5478,6 +5492,8 @@ static int ds5_probe(struct i2c_client *c, const struct i2c_device_id *id)
 	if (rec_state == 0x201) {
 		dev_info(&c->dev, "%s(): D4XX recovery state\n", __func__);
 		state->dfu_dev.dfu_state_flag = DS5_DFU_RECOVERY;
+		/* Override I2C drvdata */
+		i2c_set_clientdata(c, state);
```

Comment:
What does this function does?
Can you explain the relocation?

Reply:
Used in remove function L5552